kolibri/core/hooks.py

```diff
@@ -0,0 +1,40 @@
+"""
+Kolibri Core hooks
+------------------
+
+WIP! Many applications are supposed to live inside the core namespace to make
+it explicit that they are part of the core.
+
+Do we put all their hooks in one module or should each app have its own hooks
+module?
```

Comment:
Agreed on above - the higher-level architecture of how the plugin system works should live in core.

A perhaps more controversial but related idea:
* either - don't have a "core plugin" at all. Having a core plugin introduces special cases that only apply to it.
* or - move the core plugin to the plugins directory to be with all its friends.